<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keyBoxForm</class>
 <widget class="QWidget" name="keyBoxForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>101</width>
    <height>64</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background:		rgb(60, 60, 60);
	border: 3px solid rgb(0,174,239);
}

/* ---------- QLabels ------------*/
QLabel
{
	background:		none;
	color: 				white;
	font: 				10pt &quot;Corbel&quot;;
	border:				none;
}

QLabel:hover
{
	border:				none;
}

/*---------- QPushButton ---------*/
QToolButton
{
	background: none;
	border: none;
	image: url(:/resources/openkeywindow1_inactive.png);
}

QToolButton:pressed
{
	background: none;
	border: none;
	image: url(:/resources/openkeywindow1_active.png);
}

/* ---------- QLineEdit ------------*/
QLineEdit
{
	font: 9pt &quot;Droid Sans Mono&quot;;
	background: rgb(40, 40, 40);
	border: 2px solid rgb(0,174,239);
	border-radius: 0px;
	color: white;
	padding-left: 2px;
	padding-bottom: 2px;
}

QLineEdit:focus
{
	border: 2px solid white;
	outline: none;
}

/*-------QToolTip-------*/
QToolTip
{
	color: rgb(255, 255, 255);
	background: rgba(40, 40, 40, 140);
	border: 1px solid rgb(120, 120, 120);
	padding: 2px;
}</string>
  </property>
  <widget class="QFrame" name="keyBoxFrame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>101</width>
     <height>64</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QToolButton" name="keyBackground">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>101</width>
      <height>64</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Click this key box to&lt;br&gt; select it for copy/paste functionality</string>
    </property>
    <property name="styleSheet">
     <string notr="true">image: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="openWindow">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>13</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Futura</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="toolTip">
     <string>Open this key's modulation window.</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="keyName">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>21</y>
      <width>40</width>
      <height>26</height>
     </rect>
    </property>
    <property name="toolTip">
     <string notr="true">Key Name&lt;p&gt;Set the name of this key.</string>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit
{
	padding-top: 1px;
}

QToolTip
{
	width: 45px;
}</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>4</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
